EIPS/eip-3.md

```diff
@@ -42,6 +42,8 @@ The opcode `CALLDEPTH` should return the remaining call stack depth. A value of
 
 The actual call stack depth, as well as the call stack depth limit, are present in the EVM during execution, but just not available within the EVM. The implementation should be fairly simple and would provide a cheap and way to protect against call stack attacks.
 
-# Implementation
+# Abandoned
```

Comment:
Instead of this non-standard section, it would be better than for @holiman just to approve this PR.

Reply:
@axic   Thanks for the feedback.  Was wondering about that.  Can you point me to a definition of what is and isn't a "standard section"?  And could you provide a bit more info about exactly what @holiman should approve, so I can improve this pull request?  Should I just change line 45 back to # Implementation, yet leave that section detailing about how it is abandoned?

Reply:
@brentallsop EIP-1 lists the common sections. I am not sure there is a need for a section called "Abandoned" with the content you have here.

Reply:
Yeah, just remove this section, and I'll approve the PR